sympy/physics/control/tests/test_lti.py

```diff
@@ -170,8 +170,8 @@ def test_TransferFunction_functions():
     tf5 = TransferFunction(s**2 + 1, 1 - s, s)
     expect2 = TransferFunction((s**2 + 1)**3, (1 - s)**3, s)
     expect1 = TransferFunction((p + 4)**2, (p - 3)**2, p)
-    assert tf4*tf4 == tf4**2 == pow(tf4, 2) == expect1
-    assert tf5*tf5*tf5 == tf5**3 == pow(tf5, 3) == expect2
```

Comment:
Any reason for removing these tests?

Reply:
I'm not removing all of it. `tf4*tf4` will not be equal to `tf**2`. Although, we'll see if `(tf4*tf4).doit()` is equal to `tf**2`!